vendor/bat-native-ads/include/bat/ads/ads.h

```diff
@@ -94,9 +94,14 @@ class ADS_EXPORT Ads {
   // Should be called when a page has loaded in a browser tab, and the HTML is
   // available for analysis
   virtual void OnPageLoaded(
+      const std::string& original_url,
       const std::string& url,
       const std::string& html) = 0;
 
+  // Should be called when a push state history has changed
+  virtual void OnPushState(

```

Comment:
why does the ads library care whether it was `pushState` or not?

Reply:
Please see DM